ClangSharpPInvokeGenerator/CursorWriter.cs

```diff
@@ -971,7 +980,7 @@ private void Visit(Cursor cursor, Cursor parent)
 
             foreach (var child in cursor.Children)
             {
-                Debug.Assert(_visitedCursors.Contains(child));
+                Debug.Assert(_visitedCursors.Contains(child) || !child.IsFromMainFile);
```

Comment:
I am going to clean up, in a future PR, the `CursorWriter` to no longer silently fail to process things either and will likely set up a way to report `Diagnostics` so that we can tell the user what we failed to process.